src/mappings/StakeChanged.ts

```diff
@@ -14,3 +15,43 @@ export async function handleBonded(event: SubstrateEvent): Promise<void> {
 
     await element.save()
 }
+
+export async function handleUnbonded(event: SubstrateEvent): Promise<void> {
+    const {event: {data: [stash, amount]}} = event;
+
+    const element = new StakeChange(eventId(event));
+    element.timestamp = timestamp(event.extrinsic.block)
+    element.address = stash.toString()
+    element.amount = (-(amount as Balance)).toString()
```

Comment:
Let's save positive numbers and live the sign in the business logic. For example, remember that we are using BigUInt for values from blockchain on iOS.